core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -560,14 +560,16 @@ object ConsumerGroupCommand extends Logging {
       val groupOffsets = TreeMap[String, (Option[String], Option[Seq[PartitionAssignmentState]])]() ++ (for ((groupId, consumerGroup) <- consumerGroups) yield {
         val state = consumerGroup.state
         val committedOffsets = getCommittedOffsets(groupId)
+        // The admin client returns `null` as a value to indicate that there is not committed offset for a partition. The following getPartitionOffset function seeks to avoid NullPointerException by filtering out those null values.
```

Comment:
nit: I think that we can shorten the comment a bit. `The admin client returns `null` as a value to indicate that there is not committed offset for a partition.` seems sufficient to me.